UITests/LandscapeToolbarTests.swift

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import WebKit
+import UIKit
+
+class LandscapeToolbarTests: KIFTestCase, UITextFieldDelegate {
```

Comment:
I suggest naming the class `ToolbarTests` so we can add other (i.e., portrait) tests to this class without having a separate `PortraitToolbarTests`.